src/layouts/Footer.tsx

```diff
@@ -0,0 +1,11 @@
+import classes from './../assets/styles/Layout.module.css';
+
+const Footer = () => {
+    return(
+        <div className={classes.Footer}>
```

Comment:
Nie wiem jak z semantycznością takiego zapisu ale chyba nie powinno to tak wyglądać. Powinien być <footer></footer>, żeby semantyka HTML'a się zgadzała